study_export.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+import json
+from argparse import ArgumentParser
+from functools import cached_property
+
+import requests
+
+from ebi_eva_common_pyutils.metadata_utils import get_metadata_connection_handle
+from ebi_eva_common_pyutils.pg_utils import get_all_results_for_query
+
+
+class StudyExport:
+    """Abstract class to provide basic function for exporting studies to the JSON format"""
+    @staticmethod
+    def get_fields(all_fields):
+        return [
+            {'name': f, 'value': all_fields[f]} for f in all_fields
+        ]
+
+    @staticmethod
+    def get_entry(fields, cross_references=None, hierarchical_field=None):
```

Comment:
This method looks like it is actually "setting" the entry to the values that were passed in and returns that entry.

Reply:
apologies I correct this in the [wrong branch](https://github.com/EBIvariation/eva-ebi-search-export/pull/3)